cmd/iam.go

```diff
@@ -208,7 +216,7 @@ func (sys *IAMSys) Load(ctx context.Context, firstTime bool) error {
 	atomic.StoreUint64(&sys.LastRefreshTimeUnixNano, uint64(loadStartTime.Add(loadDuration).UnixNano()))
 	atomic.AddUint64(&sys.TotalRefreshSuccesses, 1)
 
-	if !globalSiteReplicatorCred.IsValid() {
+	if globalSiteReplicationSys.isEnabled() && !globalSiteReplicatorCred.IsValid() {
```

Comment:
let's avoid this globalSiteReplicationSys.isEnabled() for now. Currently there is a race between loading  site replication config and IAM